auth/auth.go

```diff
@@ -22,6 +22,20 @@ func LoadUserManager(authConfig evergreen.AuthConfig) (um gimlet.UserManager, su
 		}
 		return manager, true, nil
 	}
+	makeOktaManager := func() (gimlet.UserManager, bool, error) {
+		var evgURL, loginDomain string
+		env := evergreen.GetEnvironment()
+		if env != nil {
```

Comment:
Is it ever legitimate to call `NewOktaUserManager` if `env != nil`?

Reply:
Probably not, but I copy-pasted it from the github manager code. Since it seems like `LoadUserManager` depends on other things in `*evergreen.Settings`, I just changed it to pass in the settings rather than just the auth config so we don't have to use `GetEnvironment()`.